### DISCUSSION TITLE
Network Tweak for Winuntil

### DISCUSSION THREAD

**Discussion by Vexisus** [ChrisTitusTech/winutil#1529] [Ideas]
hello Chris. 

i was wondering if there any possible to at add a network tweak to reduce latency and lower ping and or like a better optimize network for gaming and better experience for both Wifi and Cable. i know there already a dns tweak in winutil

 i wanted to ask if it possible. well tried to.

regard

Vexisus

**Comment by definedfate** [comment]
You cannot lower network latency via tweaks. Whatever you see online is placebo and likely to make things worse.
